src/driver/express/ExpressDriver.ts

```diff
@@ -275,29 +277,28 @@ export class ExpressDriver extends BaseDriver implements Driver {
                 }
                 options.next();
             });
-
-        } else if (result !== undefined || action.undefinedResultCode) { // send regular result
-            if (result === null || (result === undefined && action.undefinedResultCode)) {
-                if (result === null && !action.nullResultCode) {
-                    options.response.status(204);
-                }
-
-                if (action.isJsonTyped) {
-                    options.response.json();
-                } else {
-                    options.response.send();
-                }
-                options.next();
+        } 
+        else if (result != null) { // send regular result
```

Comment:
`result != null` is an equivalent for `result !== null || result !== undefined`
I should change the order of the checks, so the last `else` would be for `send regular result`